style.css

```diff
@@ -0,0 +1,501 @@
+@font-face {
+  font-family: "nyt-imperial";
+  src:
+    url("https://g1.nyt.com/fonts/family/imperial/imperial-normal-400.2531995fefd3b997f9c4d564ebe89268.woff2")
+    format("woff2");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: OldeEnglish;
+  src: url(fonts/OldeEnglish.ttf);
+}
+
+@font-face {
+  font-family: cheltenham-small-normal-700;
+  src: url(fonts/cheltenham-small-normal-700.ttf);
+}
+
+@font-face {
+  font-family: franklin-normal-700;
+  src: url(fonts/franklin-normal-700.ttf);
+}
+
+@font-face {
+  font-family: cheltenham-italic-700;
+  src: url(fonts/cheltenham-italic-700.ttf);
+}
+
+@font-face {
+  font-family: franklin-normal-500;
+  src: url(fonts/franklin-normal-500.ttf);
+}
+
+@font-face {
+  font-family: imperial-italic-500;
+  src: url(fonts/imperial-italic-500.ttf);
+}
+
+@font-face {
+  font-family: CenturyOldStyleStdRegular;
+  src: url(fonts/CenturyOldStyleStdRegular.otf);
+}
+
+@font-face {
+  font-family: imperial-normal-700;
+  src: url(fonts/imperial-normal-700.ttf);
+}
+
+@font-face {
+  font-family: cheltenham-normal-700;
+  src: url(fonts/cheltenham-normal-700.ttf);
+}
+
+@font-face {
+  font-family: cheltenham-normal-400;
+  src: url(fonts/cheltenham-normal-400.ttf);
+}
+
+@font-face {
+  font-family: franklin-normal-300;
+  src: url(fonts/franklin-normal-300.ttf);
+}
+
+@font-face {
+  font-family: franklin-normal-600;
+  src: url(fonts/franklin-normal-600.ttf);
+}
+
+@font-face {
+  font-family: cheltenham-normal-500;
+  src: url(fonts/cheltenham-normal-500.ttf);
+}
+
+body {
+  font-size: 1.1rem;
+  line-height: 1.6em;
+}
+
+.title-container h1 {
+  font-family: OldeEnglish, Arial, sans-serif;
+  margin-bottom: 20px;
+  font-size: 29px;
+  line-height: 16px;
+  cursor: pointer;
+}
```

Comment:
- [ ] `cursor: pointer` is repeated 8 times. Please follow the basic CSS practices. ([article for support](https://speckyboy.com/good-bad-css-practices/))

- **PLEASE DO IT WITH ALL OTHER CSS FILES**